components/src/md3/ChannelValue/ChannelValue.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useCallback } from 'react';
 import { View, StyleSheet, ViewProps, ViewStyle, StyleProp, TextStyle, I18nManager } from 'react-native';
 import { MD3Theme, Text, useTheme } from 'react-native-paper';
 import { $DeepPartial } from '@callstack/react-theme-provider';
-import { Icon } from '../Icon';
+import { Icon } from '../icon';
```

Comment:
I just noticed the build is outputting the icon folder with a lowercase. Not sure why this is happening, but if this change works then we can roll with it.